packages/logger/test/unit/utils/json.test.ts

```diff
@@ -27,13 +25,13 @@ describe("Json helper", () => {
 
       // Functions
       // eslint-disable-next-line @typescript-eslint/no-empty-function
-      {id: "function", arg: function () {}, json: "function () { }"},
+      {id: "function", arg: function () {}, json: "function() {\n      }"},
```

Comment:
I don't really get from where the line breaks are coming

Function toString() does not seem to add them
https://github.com/ChainSafe/lodestar/blob/2530fae45382733f48124fb0afb28084ca4ad60f/packages/logger/src/utils/json.ts#L25-L26

See output
```node
> (function() {}).toString()
'function() {}'
```

Or is this related to what you mentioned here https://github.com/ChainSafe/lodestar/pull/6220#discussion_r1433856180?

Reply:
Yes, the stub earlier was removing the new line break from the output.

Reply:
But I still don't get why there would be a new line in the middle of the function, what's adding that?

Reply:
When `vitest` transpile the TS code to JS the `esbuild/rollup` add a new line at the function definition, that translates to new line when we do the `function.string()`. 

```js
const arg = function () {
};
console.log(arg.toString());
```

I could not find the direct fix tin the Rollup config for it, and fixing the test fixture is not destructive in this case, so left it like that.

Reply:
I also don't think it is worth to try and find a fix for it, just good to know why these new lines are there